security/accesscontrol.rst

```diff
@@ -215,6 +227,40 @@ A valid access token for Firely Server at minimum will have:
 
 .. warning:: Firely Server will not enforce any access control for resources outside of the specified compartment. Some compartment definitions do not include crucial resource types like 'Patient' or their corresponding resource type, i.e. all resources of this type regardless of any claims in the access token will be returned if requested. Please use this feature with caution! Additional custom access control is highly recommended.
 
+.. _feature_accesscontrol_aad:
+
+Azure Active Directory
+----------------------
+
+Azure Active Directory (v2.0) does not allow to define a scope with ``/`` (forward slash) in it, which is not compatible with the structure of a `SMART on FHIR scope <http://hl7.org/fhir/smart-app-launch/1.0.0/scopes-and-launch-context/index.html>`_. 
+Therefor when you use AAD to provide SMART on FHIR scopes to Firely Server, you need to take following steps
+
+1. In a SMART scope, use another character (for instance ``-``) instead of ``/``. For example:
+
+  * ``user/*.read`` becomes ``user-*.read``
+  * ``user/*.write`` becomes ``user-*.write``
+  * ``patient/Observation.r`` becomes ``patient-Observation.r``
+  
+  If the used character (for instance ``-``) is already in your SMART scope, then you can use ``\`` (backward slash) to escape it.
+  
+  * ``patient/Observation.r?_id=Id-With-Dashes`` becomes ``patient-Observation.r?_id=Id\-With\-Dashes``
+
+  If a ``\`` (backward slash) is already in your SMART scope, then you can escape it with another ``\``.
+
+  * ``patient/Observation.r?_id=Id\With\BackwardSlash`` becomes ``patient-Observation.r?_id=Id\\With\\BackwardSlash`` 
+
+2. Configure Firely Server which character is used in Step 1, then Firely Server will generate a proper `SMART on FHIR scope <http://hl7.org/fhir/smart-app-launch/1.0.0/scopes-and-launch-context/index.html>`_ and handle the request further. This can be configured via setting ``AccessTokenScopeReplace``. 
+
+For the first step above, instead of doing it manually, you can deploy `SMART on FHIR AAD Proxy <https://github.com/azure-smart-health/smart-on-fhir-aad-proxy>`_ to Azure, which helps you to replace ``/`` to ``-`` in a SMART scope.
```

Comment:
```suggestion
For the first step above, instead of doing it manually, you can deploy `SMART on FHIR AAD Proxy <https://github.com/azure-smart-health/smart-on-fhir-aad-proxy>`_ to Azure, which helps you to replace ``/`` to ``-`` in a SMART scope when you request your access token.
```